datasets/census_bureau_acs/_images/run_csv_transform_kub/csv_transform.py

```diff
@@ -0,0 +1,550 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import datetime
+import json
+import logging
+import os
+import pathlib
+import time
+import typing
+
+import numpy as np
+import pandas as pd
+import requests
+from google.cloud import storage
+
+
+def main(
+    source_url: str,
+    year_report: str,
+    api_naming_convention: str,
+    target_file: pathlib.Path,
+    target_gcs_bucket: str,
+    target_gcs_path: str,
+    headers: typing.List[str],
+    rename_mappings: dict,
+    pipeline_name: str,
+    geography: str,
+    report_level: str,
+    concat_col: typing.List[str],
+) -> None:
+
+    logging.info(
+        f"ACS {pipeline_name} process started at "
+        + str(datetime.datetime.now().strftime("%Y-%m-%d %H:%M:%S"))
+    )
+
+    logging.info("Creating 'files' folder")
+    pathlib.Path("./files").mkdir(parents=True, exist_ok=True)
+
+    group_id = {
```

Comment:
Let's put this in a `.json` file in the same directory as this file, and then load it

```
# File is group_ids.json
group_id = json.load("group_ids.json")
```

Reply:
created separate json file